src/Shared/UnitTests/XmakeAttributes_Tests.cs

```diff
@@ -2,6 +2,8 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using Microsoft.Build.Shared;
+
```

Comment:
nit whitespace

Reply:
This is actually my preferred style:

1. Block for `System.` / BCL usings
2. Block for local-repo usings
3. Block for third-party usings